lib/constants/enums.ts

```diff
@@ -2,6 +2,7 @@
 export enum SwapClientType {
   Lnd,
   Raiden,
+  Connext,
```

Comment:
Now that we have more than 1 swap client we'll need to modify `SwapRecovery` L83 to also save the recovered preimages to memory. This is used when the payment succeeded, but xud managed to crash before using the preimage to claim the second payment.

Reply:
Is this still relevant if each asset only has one SwapClient?